src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -325,6 +324,48 @@ await supersededPublications
                 .OnSuccess(publication => _mapper.Map<PublicationViewModel>(publication));
         }
 
+        public async Task<Either<ActionResult, ExternalMethodologyViewModel>> GetExternalMethodology(Guid publicationId)
+        {
+            return await _persistenceHelper
+                .CheckEntityExists<Publication>(publicationId)
+                .OnSuccessDo(_userService.CheckCanViewPublication)
+                .OnSuccess(publication => publication.ExternalMethodology != null
+                    ? _mapper.Map<ExternalMethodologyViewModel>(publication.ExternalMethodology)
+                    : NotFound<ExternalMethodologyViewModel>());
+        }
+
+        public async Task<Either<ActionResult, ExternalMethodologyViewModel>> UpdateExternalMethodology(
+            Guid publicationId, ExternalMethodology updatedExternalMethodology)
+        {
+            return await _persistenceHelper
+                .CheckEntityExists<Publication>(publicationId)
+                .OnSuccessDo(_userService.CheckCanUpdatePublication)
+                .OnSuccess(publication =>
+                {
+                    _context.Update(publication);
+                    publication.ExternalMethodology = updatedExternalMethodology;
```

Comment:
Note that given we've allowed a user of the API to potentially update ANY field of the ExternalMethodology here, that they could potentially do something nefarious here, like amend the JSON of the POST to alter fields of the ExternalMethodology that we wouldn't intend them to alter, and that the front end wouldn't let them alter. 

Safer (albeit with more boilerplate) would be to have the Controller endpont accept an `UpdateExternalMethodologyRequest` / `ExternalMethodologyUpdateRequest` that includes just the fields that we want a user to be able to update, then manually apply those field updates to either an existing or a new ExternalMethodology object on this Publication.

Reply:
Created `ExternalMethodologySaveRequest` record - basically used in the same vein as "PublicationSaveRequest`.